### ISSUE TITLE
npm ERR! Please try running this command again as root/Administrator.

### ISSUE THREAD

**Issue by AltRoot17** [npm/npm#19110]
Im all ready read this error for the solutions, but its not work for me , how to fix it 

```
D:\Mobile Application\Ionic\myApp>npm install ion2-calendar moment --save
npm ERR! path D:\Mobile Application\Ionic\myApp\node_modules\fsevents\node_modules
npm ERR! code EPERM
npm ERR! errno -4048
npm ERR! syscall scandir
npm ERR! Error: EPERM: operation not permitted, scandir 'D:\Mobile Application\Ionic\myApp\node_modules\fsevents\node_modules'
npm ERR!  { Error: EPERM: operation not permitted, scandir 'D:\Mobile Application\Ionic\myApp\node_modules\fsevents\node_modules'
npm ERR!   stack: 'Error: EPERM: operation not permitted, scandir \'D:\\Mobile Application\\Ionic\\myApp\\node_modules\\fsevents\\node_modules\'',
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'scandir',
npm ERR!   path: 'D:\\Mobile Application\\Ionic\\myApp\\node_modules\\fsevents\\node_modules' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Hp Probook\AppData\Roaming\npm-cache\_logs\2017-11-09T08_23_38_831Z-debug.log


```

**Comment by ghost** [comment]
this happen when you're logged in as non-admin, usually [takeown](https://technet.microsoft.com/en-us/library/cc753024(v=ws.11).aspx) can resolve the issue.

try `takeown /f <File_name> /r`.

**Comment by MAQ7** [comment]
I have the same issue although I am using admin account and I run the command shell as Admin!!

**Comment by mahgolfa** [comment]
simply run sudo before : 
sudo npm install -g create-react-native-app
and it will ask u for u password, the result should be something like this
  ╭─────────────────────────────────────╮
   │                                     │
   │   Update available 5.6.0 → 6.1.0    │
   │     Run npm i -g npm to update      │
   │                                     │
   ╰─────────────────────────────────────╯
run sudo npm i -g npm
that's it :)
